reproducibility-with-guix.org

```diff
@@ -376,12 +397,12 @@ and in a next step into
 [[file:data-program-environment.png]]
 We can go on and divide the environment into operating system, development tools, and application software, for example, but the further we go in decomposing the input to a computation, the more arbitrary it gets.
 
-From this point of view, a software's dependencies consist of everything required to run it in addition to its source code. For a Guix package, the dependencies are thus
+From this point of view, a software's dependencies consist of everything required to run it in addition to its source code. For a Guix package, the dependencies are thus,
 - its inputs
 - the build system arguments
 - the build system itself
-- Guix
-- the GNU/Linux operating system
+- Guix (commit)
+- the GNU/Linux operating system (kernel).
```

Comment:
It's a bit more than the kernel. The file system, for example. Maybe more - I am not sure what the smallest dependency list is. In practice, installing a binary distribution of Guix requires a shell plus some utilities such as tar. Maybe it could be done with less.

Reply:
Maybe replace `(kernel)` by `(kernel, filesystem, etc.)` or `(low layer close to hardware)`.

To me, the term `GNU/Linux operating system` is confusing because I am not sure about what we are talking about.